commonknowledge/meep-intelligence-hub
# Feature/meep 183 add action network support just treatment medact

## Description
Adds ActionNetwork as a Data Source. Also updates the Data Source front-end UX to use pre-set field names for certain CRMs (e.g. for ActionNetwork, the first name field is always `given_name`.

## Motivation and Context
https://linear.app/commonknowledge/issue/MAP-60/action-network-integration

## How Can It Be Tested?
Add the Common Knowledge Main Test Group to Mapped, check data can be imported and updated.

## How Will This Be Deployed?
Normal method, no extra env vars required.

## Screenshots (if appropriate):

## Types of changes


## Checklist:


- My change requires a change to the documentation.
- I've updated the documentation accordingly.
